src/app/projects/components/project-settings/components/embeddings/embeddings.component.ts

```diff
@@ -65,18 +86,43 @@ export class EmbeddingsComponent implements OnInit, OnDestroy, OnChanges {
       }));
   }
 
+  prepareEmbeddingPlatforms() {
+    if (this.organization.gdprCompliant && this.isManaged) {
+      this.embeddingPlatforms = this.embeddingPlatforms.filter((platform) => platform.gdprCompliant === true);
+    }
+    this.embeddingPlatforms.forEach((platform: EmbeddingPlatform) => {
+      platform.name = platformNamesDict[platform.platform];
+    });
+  }
+
   checkForceHiddenHandles() {
-    const granularity = this.settingModals.embedding.create.embeddingCreationFormGroup.get('granularity').value;
-    const attId = this.settingModals.embedding.create.embeddingCreationFormGroup.get('targetAttribute').value;
-    const suggestionList = this.embeddingHandles[attId];
+    const form = this.settingModals.embedding.create.embeddingCreationFormGroup;
+    const granularity = form.get('granularity').value;
+    const attId = form.get('targetAttribute').value;
+    const platform = form.get('platform').value;
+    let suggestionList = this.embeddingHandlesCopy[attId];
+    if (platform == PlatformType.PYTHON) {
+      suggestionList = suggestionList.filter(e => e.configString == 'bag-of-words' || e.configString == 'bag-of-characters' || e.configString == 'tf-idf');
+      this.embeddingHandles[attId] = suggestionList;
+    } else if (platform == PlatformType.OPEN_AI) {
+      suggestionList = suggestionList.filter(e => e.configString == 'text-embedding-ada-002');
+      this.embeddingHandles[attId] = suggestionList;
+    } else if (platform == PlatformType.HUGGING_FACE) {
+      this.embeddingHandles[attId] = this.embeddingHandlesCopy[attId];
+      this.checkModelDownloaded();
+    }
```

Comment:
these are filtered by a hardcoded config string. so if a new config string is added this result lin faulty values. That's why I mentioned that we should add a platformtype in the suggestions json from the backend so it's easier to filter :)